aws-wildfly/src/test/java/aws/cloudformation/archiveDeploy/ec2Domain/suitetests/AwsDomainEc2SecondDeployOtherGroupTest.java

```diff
@@ -0,0 +1,73 @@
+package aws.cloudformation.archiveDeploy.ec2Domain.suitetests;
+
+
+import aws.cloudformation.archiveDeploy.ec2Domain.DomainEc2DeploySuiteTests;
+import sunstone.annotation.Deployment;
+import sunstone.annotation.DomainMode;
+import sunstone.annotation.OperatingMode;
+import sunstone.annotation.Parameter;
+import sunstone.annotation.WildFly;
+import sunstone.aws.annotation.AwsEc2Instance;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.assertj.core.api.Assertions;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.jupiter.api.Test;
+import sunstone.aws.annotation.WithAwsCfTemplate;
+import sunstone.aws.impl.AwsWFLogger;
+import sunstone.inject.Hostname;
+
+import java.io.IOException;
+
+import static aws.cloudformation.AwsTestConstants.region;
+
+/**
+ * Deploy only to other-server-group, app shouldn't be deployed to main-server-group
+ */
+@WithAwsCfTemplate(parameters = {
+        @Parameter(k = "instanceName", v = DomainEc2DeploySuiteTests.suiteInstanceName)
+},
+        template = "sunstone/aws/cloudformation/eapDomain.yaml", region = region, perSuite = true)
+public class AwsDomainEc2SecondDeployOtherGroupTest {
+    @AwsEc2Instance(nameTag = DomainEc2DeploySuiteTests.suiteInstanceName, region = region)
+    @WildFly(mode = OperatingMode.DOMAIN)
+    Hostname hostname;
+
+    @Deployment(name = "testapp.war")
+    @AwsEc2Instance(nameTag = DomainEc2DeploySuiteTests.suiteInstanceName, region = region)
+    @WildFly(mode = OperatingMode.DOMAIN, domain = @DomainMode(serverList = "other-server-group"))
+    static WebArchive deploy() {
+        return ShrinkWrap.create(WebArchive.class)
+                .addAsWebResource(new StringAsset("Hello World"), "index.jsp");
+    }
+
+    @Test
+    public void test() throws IOException {
```

Comment:
Can the test name reflect the fact that deployment from the `AwsDomainEc2FirstDeployAllGroupsTest` should be undeployed?